systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -360,7 +360,7 @@ void testTlsExternalUser() {
                 KafkaNodePoolTemplates.controllerPool(testStorage.getNamespaceName(), testStorage.getControllerPoolName(), testStorage.getClusterName(), 1).build()
             )
         );
-        resourceManager.createResourceWithWait(KafkaTemplates.kafkaEphemeral(testStorage.getClusterName(), 1, 1)
+        resourceManager.createResourceWithWait(KafkaTemplates.kafkaPersistent(testStorage.getClusterName(), 1, 1)
```

Comment:
I guess that on some places the change between ephemeral and persistent storage doesn't make much sense (at least from MPOV).
It's not about having some problem with resources or something
I just thing that for the tests where no rolling update is happening or we are not depended on the storage, we can use the ephemeral storage

Or maybe I'm wrong 🤷

Reply:
In general, if you don't do rolling updates the likely hood of something going wrong with ephemeral storage is of course much smaller. It might still happen if something restarts due to some issue etc.

Reply:
yeah you both have a point here, 
So as was agreed if there is for some reason (too many resources) only 1 replica , storage is changed to persistent, otherwise I am keeping it as it is so far (I am removing some of increment of replicas due to actually hitting one resource limitation).